packages/opc-header/README.md

```diff
@@ -0,0 +1,83 @@
+# This is opc-header 👋
+
+Web component based on lit-element, it can be used as a other element of HTML example <opc-header> </opc-header>
+
+https://docs.google.com/document/d/1PoxpPsret2njugkAG_YBmlS3ZNkDd0MTY-JZKDbesrQ/edit#heading=h.w85tasqfwpfc
+
+## Install
+
+```sh
+npm i  @one-platform/opc-header 
+```
+
+## Usage
+
+import '@one-platform/opc-header/dist/opc-header';
+
+<opc-header header-name="Outages"></opc-header>
```

Comment:
follow the standard for attribute names

Reply:
@diwanshi you mean it should not have -  or something like this headername ?

Reply:
@hybridx can you please guide @anjneeksharma

Reply:
Ack @diwanshi